File: codes/python/p097.py
# 
# Solution to Project Euler problem 97
# Copyright (c) Sankha Saha. All rights reserved.
# 
#  
#   
# 


def compute():
	MOD = 10**10
	ans = (28433 * pow(2, 7830457, MOD) + 1) % MOD
	return str(ans)


if __name__ == "__main__":
	print(compute())

File: codes/Haskell/p097.hs
{- 
 - Solution to Project Euler problem 97
 - Copyright (c) Sankha Saha. All rights reserved.
 - 
 -  
 -   
 -}


main = putStrLn (show ans)
ans = mod (28433 * 2^7830457 + 1) (10^10) :: Integer
